cmake/projects/OpenSSL/schemes/url_sha1_openssl_windows_1_1_plus.cmake.in

```diff
@@ -64,21 +66,35 @@ else()
   set(opt "VC-WIN32")
 endif()
 
-if("@MSVC@")
-  # Logging as Workaround for VS_UNICODE_OUTPUT issue:
-  # https://public.kitware.com/Bug/view.php?id=14266
-  set(log_opts LOG_CONFIGURE 1)
+if(ASM_SUPPORT)
+  hunter_add_package(NASM)
+  configure_file(
+    "@HUNTER_PACKAGE_SETUP_DIR@/externals/nasm_path_configure.cmake.in"
+    "@HUNTER_PACKAGE_BUILD_DIR@/nasm_path_configure.cmake"
+    @ONLY
+  )
 else()
-  set(log_opts "")
+  list(APPEND opt "no-asm")
 endif()
 
+#if("@MSVC@")
+  # Logging as Workaround for VS_UNICODE_OUTPUT issue:
+  # https://public.kitware.com/Bug/view.php?id=14266
+#  set(log_opts LOG_CONFIGURE 1)
+#else()
+#  set(log_opts "")
+#endif()
+
 if (BUILD_SHARED_LIBS)
   set(shared "shared")
 else()
   set(shared "no-shared")
 endif()
 
 set(openssl_dir "@HUNTER_PACKAGE_INSTALL_PREFIX@/ssl")
+set(openssl_install_dir "@HUNTER_PACKAGE_INSTALL_PREFIX@")
```

Comment:
Move it before `configure_file` so `@openssl_install_dir@` can be used there. Add a `Used variables` comment about dependency, see example:
* https://github.com/ruslo/hunter/blob/v0.23.50/cmake/projects/Qt/schemes/url_sha1_qt.cmake.in#L354-L368

Reply:
configure is now working... I think we need to move nmake too ...